container/options.go

```diff
@@ -216,6 +242,19 @@ func SplitPercent(p int) SplitOption {
 	})
 }
 
+// SplitFixed sets the size as a fixed value of the available space.
+// FIXME: More comments
```

Comment:
Did we want to remove the FIXME and add comments? We could take what was written for SplitPercent and adjust accordingly.

Reply:
Forgot about this one, was a comment for myself before I pushed changes, but slipped my mind.
I'll add comments that are similar to SplitPercent. :+1: